getting-started/proc-add-source-control-rhdh-catalog.adoc

```diff
@@ -183,16 +206,41 @@ You can enable GitHub discoverability for your components in {product-short}, su
 
 .Procedure
 
-. In Red Hat Openshift, go to the *Helm* tab and upgrade the _Developer Hub_ release.
-. Under *global → Dynamic plugins configuration → List of dynamic plugins that should be installed in the backstage application. → Package specification of the dynamic plugin to install. It should be usable by the `npm pack` command.*, add the following value:
+. In the *Developer* perspective of the OpenShift web console, go to the *Helm* tab.
```

Comment:
Using _OpenShift_ alone is not an approved by brand. PR #135 creates [attributes for approved RHOCP names](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/135/files#diff-0f7e8a8fb3fdb835a98a8a937c0c87986260380b80b77a1f84f8ca529be430a2R19), but it has not been merged yet.

Reply:
I agree. i was also thinking to create an attribute. but, this change was global. good to know that you are adressing this issue :)